# origin: Debian (joshk)
# cset: n/a
# inclusion: submitted (2004-09-17)
# description: support GNU ld output in ver_linux
# revision date: 2004-09-05

--- a/scripts/ver_linux	2004-09-05 01:31:23.000000000 -0700
+++ b/scripts/ver_linux	2004-09-05 01:31:47.000000000 -0700
@@ -22,7 +22,8 @@
       '/GNU Make/{print "Gnu make              ",$NF}'
 
 ld -v 2>&1 | awk -F\) '{print $1}' | awk \
-      '/BFD/{print "binutils              ",$NF}'
+      '/BFD/{print "binutils              ",$NF}
+       /^GNU/{print "binutils              ",$4}'
 
 fdformat --version | awk -F\- '{print "util-linux            ", $NF}'
 
